README.md

```diff
@@ -1 +1,4 @@
-# TestRepo
```

Comment:
The following issues were found in the file  
Issue 1: The root element '# TestRepo' is missing a closing tag.